src/lambda/update-syllabus/utils.py

```diff
@@ -0,0 +1,72 @@
+from const import *
+
+class course():
```

Comment:
classes should have the first letter capitalized. Also you do not need brackets
```suggestion
class Course:
```

Reply:
Fixed.